### ISSUE TITLE
4.1 & 4.1.1 Cannot Launch

### ISSUE THREAD

**Issue by Phoenixkaze** [godotengine/godot#80078]
### Godot version

4.1 & 4.1.1 stable (including .NET builds)

### System information

Windows 11 22H2

### Issue description

I tried to use 4.1.1 on my Windows 11 device but it did not launch. Nothing shows up when I run the executable. Nothing shows in console.

![image](https://github.com/godotengine/godot/assets/28686529/caf52094-e74c-4a1d-a720-527a75213b81)

The window shows up and exits seconds later. No error messages or dialogs.

I also tried 4.1 (including .NET builds). Same issue. 4.0 works well.

### Steps to reproduce

Download the builds and try to launch.

### Minimal reproduction project

No project is needed to reproduce this.

**Comment by manimonji** [comment]
I have the same problem. My older pc can open 4.0 and 4.1, but I have another pc which can't open 4.0 and 4.1 versions.

**Comment by Phoenixkaze** [comment]
Yeah I have another pc running Windows 10. All the versions mentioned work on that but not on the Windows 11 one.

**Comment by Calinou** [comment]
@Phoenixkaze @manimonji Can you try following the same steps on 4.1 dev/beta/RCs? You can download them [here](https://downloads.tuxfamily.org/godotengine/4.1/).

Also, can you reproduce this if you configure Windows 11 to use the legacy console host by default instead of Windows Terminal?

**Comment by Phoenixkaze** [comment]
4.1 RC1 does not launch. Also reproduced with different terminal configurations. Same symptoms like those of the stable.
4.1 RC2 same.
4.1 RC3 same.

---

4.1 beta1 I can see the logo. Then the project manager window shows up. But nothing in the window. It exits seconds later.(See the image of dev4)
4.1 beta2 same as RC and stable.
4.1 beta3 same as RC and stable.

----

**4.1 dev1: seems that everything is okay.**
**4.1 dev2: seems that everything is okay.**
**4.1 dev3: seems that everything is okay.**
4.1 dev4: I can see the logo. Then the project manager window shows up. But nothing in the window. It exits seconds later.

![image](https://github.com/godotengine/godot/assets/28686529/5ee52d80-3282-4e93-97be-9c763e68767c)

**Comment by Phoenixkaze** [comment]
Just tried the 4.1.2 release. Same issue.
4.2 dev 6 works fine while dev 5 has same issue.

**Comment by akien-mga** [comment]
So 4.2 dev 6 is working fine? If so it might be using ANGLE which would work better on this hardware.

Can you confirm by launching it with the "console" exe and sharing what is written at the start of the logs?

**Comment by Phoenixkaze** [comment]
Yes 4.2 dev 6 is working fine. It printed this to console:
```
OpenGL API 3.3.0 - Build 31.0.101.4648 - Compatibility - Using Device: Intel - Intel(R) Iris(R) Xe Graphics
```
Those builds not working print nothing and just exit.

**Comment by akien-mga** [comment]
Great, closing as fixed in 4.2 then :)
